### PR TITLE
feat:added categories to video library

### PR DESCRIPTION
**Pull request by mariyasada**
added data to the categories.js file and map that category and create a category card for the home page. The home page is totally responsive.
categorycard

### CODE REVIEW

File: src/Components/HomepageCard/homepagecard.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.Category-card-container {
+  height: 450px;
```
Comment (ShreyashMahajan) [inline src/Components/HomepageCard/homepagecard.css]: you can use rem for more than 10px

File: src/Components/HomepageCard/homepagecard.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  width: 100%;
+  gap: 1rem;
+  cursor: pointer !important;
```
Comment (ShreyashMahajan) [inline src/Components/HomepageCard/homepagecard.css:5]: avoid using important, if possible

**Review by ShreyashMahajan — APPROVED** [review]
good to merge. just look at the comments which i written

**Review by prajapati-priyanka — APPROVED** [review]
Clean Code. Looks good to merge.

File: src/Components/HomepageCard/homepagecard.css (hunk 1 above)
Comment (mariyasada) [reply src/Components/HomepageCard/homepagecard.css]: Will do

**Pull request merged**